ark/segmentation/signal_extraction.py

```diff
@@ -1,31 +1,34 @@
 import numpy as np
 
 
-def positive_pixels_extraction(cell_coords, image_data, threshold=0):
+def positive_pixels_extraction(cell_coords, image_data, **kwargs):
     """Extract channel counts by summing over the number of non-zero pixels in the cell.
 
     Improves on default_extraction by not distinguishing between pixel expression values
 
     Args:
         cell_coords (numpy.ndarray): values representing pixels within one cell
         image_data (xarray.DataArray): array containing channel counts
-        threshold (int): where we want to set the cutoff for a positive pixel, default 0
+        **kwargs: arbitrary keyword arguments
 
     Returns:
         numpy.ndarray:
             Sums of counts for each channel
     """
 
+    if 'threshold' not in kwargs.keys():
```

Comment:
So I have a slight preference for the pattern that we use in deepcell for handling functions with kwargs. 

```
compute_marker_counts(extraction='total_intensity', **kwargs)
     
    if extraction == 'total_intensity':
              cell_counts = total_intensity(image_data)

    elif extraction == 'positive_pixels`:
                threshold = kwargs.get('threshold', 0)
                cell_counts = positive_pixels(image_data, threshold=threshold)

    elif extraction == 'center_weighting'
                 centroid = kwargs.get(centroid)
                  cell_counts = center_weighting(image_data, centroid=centroid)
```
The benefit of this is that we don't need the helper functions to get kwargs passed to them, their API is totally defined. This makes it less likely something will go run, and makes it more understandable to read what's happening. It also means that all the error checking happens in the top level function. 

The downside is that you need a if/else condition set up for all of the potential extraction options. I think the tradeoff is worth it, but I'm also open to your approach if you think its better

Reply:
I'd lean towards my implementation, mostly because it consolidates all necessary changes for an additional extraction to the `signal_extraction.py` file (and also because I think mapping generalized function pointers in a dictionary is cool and efficient).

I think the deepcell approach makes sense for larger functions with less, immediately understandable function bodies, where clear API instructions/docstrings are requisites.  If we end up including a complicated extraction function, I'd hop on board for that approach.

That being said, I'll probably switch to using `.get` for accessing kwargs (I didn't know it was a thing haha).